### PR TITLE
[IMP] *_*: hide country specitif fields in company form view

### PR DESCRIPTION
**Pull request by mitali-willdooit**
Description of the issue/feature this PR addresses:
<b> Task:</b> https://www.odoo.com/web?#id=2049977&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
<b> Pad:</b> https://pad.odoo.com/p/r.e20dfeb073ab9b4ba1fd559834032724

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/company.py [hunk 1]
```diff
@@ -107,6 +107,9 @@ class ResCompany(models.Model):
     # Needed in the Point of Sale
     account_default_pos_receivable_account_id = fields.Many2one('account.account', string="Default PoS Receivable Account")
 
+    # Techinal field to hide country specific fields in company form view
```
Comment (mreficent) [inline addons/account/models/company.py]:
```suggestion
    # Technical field to hide country specific fields in company form view
```

File: addons/account/models/res_config_settings.py [hunk 2]
```diff
@@ -87,6 +87,8 @@ class ResConfigSettings(models.TransientModel):
     use_invoice_terms = fields.Boolean(
         string='Default Terms & Conditions',
         config_parameter='account.use_invoice_terms')
+    # Techinal field to hide country specific fields from configuration
```
Comment (mreficent) [inline addons/account/models/res_config_settings.py]:
```suggestion
    # Technical field to hide country specific fields from configuration
```

Comment (jco-odoo) [reply addons/account/models/res_config_settings.py]: @mpa-odoo Please recheck!!!  Can you rebase, check if there are maybe other fields we forgot, ...   I think we can merge this on Monday then.  @rgo-odoo

Comment (rgo-odoo) [reply addons/account/models/res_config_settings.py]: @jco-odoo I think it should be ok now, can you check ? Thanks.

File: addons/account/models/company.py [hunk 3]
```diff
@@ -107,6 +107,9 @@ class ResCompany(models.Model):
         domain="[('internal_group', '=', 'asset'), ('internal_type', 'not in', ('receivable', 'payable')), ('reconcile', '=', True), ('company_id', '=', id)]")
     accrual_default_journal_id = fields.Many2one('account.journal', help="Journal used by default for moving the period of an entry", domain="[('type', '=', 'general')]")
 
+    # Techninal field to hide country specific fields in company form view
```
Comment (jco-odoo) [inline addons/account/models/company.py]: @rgo-odoo Is it possible to use the correct spelling everywhere?  Technical

File: addons/account/models/res_config_settings.py [hunk 4]
```diff
@@ -89,6 +89,9 @@ class ResConfigSettings(models.TransientModel):
         string='Default Terms & Conditions',
         config_parameter='account.use_invoice_terms')
 
+    # Techninal field to hide country specific fields from accounting configuration
```
Comment (jco-odoo) [inline addons/account/models/res_config_settings.py]: here too

File: addons/l10n_eu_service/models/res_config_settings.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    l10n_eu_services_eu_country = fields.Boolean('Is European country?', compute='_compute_l10n_eu_services_european_country')
+
+    @api.depends('company_id')
+    def _compute_l10n_eu_services_european_country(self):
+        self.l10n_eu_services_eu_country = bool(self.env.ref('base.europe').id in self.company_id.country_id.country_group_ids.ids)
```
Comment (jco-odoo) [inline addons/l10n_eu_service/models/res_config_settings.py]: More logical to use 'country in group' then country group in country groups?  (logical code vs debatable perf gains)

File: addons/account/models/res_config_settings.py (hunk 2 above)
Comment (jco-odoo) [reply addons/account/models/res_config_settings.py]: see review

File: addons/l10n_eu_service/models/res_config_settings.py (hunk 5 above)
Comment (rgo-odoo) [reply addons/l10n_eu_service/models/res_config_settings.py]: @jco-odoo agreed! And, I've rearranged commits as you suggested and fixed those typos. So, I think today is the day ;)

**Comment by jco-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**